filecoin-proofs/src/api/mod.rs

```diff
@@ -561,3 +530,64 @@ pub unsafe extern "C" fn get_staged_sectors(
 
     raw_ptr(response)
 }
+
+#[repr(C)]
+pub struct FFISectorClass {
+    sector_size: FFISectorSize,
+    seal_proof_partitions: FFISealProofPartitions,
+    post_proof_partitions: FFIPoStProofPartitions,
+}
+
+#[allow(non_camel_case_types)]
+#[repr(C)]
+pub enum FFISectorSize {
+    SSB_OneKiB,
+    SSB_TwoHundredFiftySixMiB,
```

Comment:
I just want to check once more that we're comfortable with a model for change in which new sizes might be inserted between existing sizes, such that — for example — the underlying enum value `1` changes significance between releases. If these values go on chain anywhere, they will be more stably interpretable through time if we use the actual sizes. (I know we've discussed and you favor this approach, just calling this aspect out explicitly one last time.)

Reply:
> might be inserted between existing sizes

Yep, I'm with you. The position in the enum isn't meaningful in either go-filecoin or rust-fil-proofs.